cookieplone/settings.py

```diff
@@ -23,12 +57,12 @@
 }
```

Comment:
I think that the earliest Volto version we support is the one compatible with Cookieplone. See https://6.docs.plone.org/volto/upgrade-guide/index.html#upgrade-18-cookieplone-label. If not, please disregard. (I couldn't comment on the correct line 54)

```
VOLTO_MIN_VERSION = "18.0.0-alpha.43"
```